react/features/analytics/middleware.js

```diff
@@ -6,8 +6,10 @@ import {
     SET_ROOM
 } from '../base/conference';
 import { SET_CONFIG } from '../base/config';
+import { LIB_WILL_INIT } from '../base/lib-jitsi-meet';
 import { SET_NETWORK_INFO } from '../base/net-info';
 import { MiddlewareRegistry } from '../base/redux';
+import logger from '../base/redux/logger';
```

Comment:
each feature should has each own logger, can you create one at `react/features/analytics/logger.js`?

Reply:
Dunno how that got in there, will change.